example/custom_domain_self_managed_certs/README.md

```diff
@@ -0,0 +1,7 @@
+# Custom Domain with Self-managed Certificates on GCP
```

Comment:
Incredibly thorough example, this is so much appreciated! Thanks @impl for always going the extra mile.